tests/scripts/github_mergebot.py

```diff
@@ -0,0 +1,552 @@
+
+        return body
+
+    def merge(self) -> None:
+        """
+        Request a merge of this PR via the GitHub API
+        """
+        url = f"pulls/{self.number}/merge"
+
+        title = self.raw["title"]
+        body = self.body_with_co_authors()
+        self.lint_commit_message(title, body)
+        logging.info(f"Full commit:\n{title}\n\n{body}")
+
+        data = {
+            "commit_title": title,
+            "commit_message": body,
+            # The SHA is necessary in case there was an update right when this
+            # script ran, GitHub will sort out who won
+            "sha": self.head_oid(),
+            "merge_method": "squash",
+        }
+        if self.dry_run:
+            logging.info(f"Dry run, would have merged with url={url} and data={js(data)}")
+            return
+
+        self.github.put(url, data=data)
+
+    def comment_can_merge(self, comment: Dict[str, Any]) -> bool:
+        """
+        Check if a comment was left by the PR author or by a committer
+        """
+        print(comment)
+        print(self.raw)
+        if comment["author"]["login"] == self.raw["author"]["login"]:
+            logging.info(f"Comment {comment} was from author and is mergable")
+            return True
+
+        if comment.get("authorAssociation", "") == "CONTRIBUTOR":
+            logging.info(f"Comment {comment} was from committer commment and is mergable")
+            return True
+
+        if comment.get("authorCanPushToRepository", False):
+            logging.info(f"Comment {comment} was from a committer review comment and is mergable")
+            return True
+
+        logging.info(f"Comment {comment} was not from author or committers and is not mergable")
+        return False
+
+    def merge_requested(self) -> bool:
+        """
+        Check if this PR has had a merge requested
+        """
+        merge_commands = [
+            "merge",
+            "merge this",
+            "merge this pr",
+        ]
+        cancel_commands = [
+            "cancel",
+            "cancel merge",
+            "cancel the merge",
+            "stop",
+            "stop merge",
+            "stop the merge",
+        ]
+
+        def parse_action(comment: Dict[str, Any]) -> Optional[str]:
+            if not self.comment_can_merge(comment):
+                return None
+
+            body = comment["body"]
+            if any(f"@tvm-bot {c}" in body for c in merge_commands):
+                return "merge"
+
+            if any(f"@tvm-bot {c}" in body for c in cancel_commands):
+                return "cancel"
+
+            return None
+
+        # Check regular comments
+        all_comments = []
+        for comment in self.raw["comments"]["nodes"]:
+            all_comments.append(comment)
+
+        # Check top-level review comments
+        for review in self.reviews():
+            all_comments.append(review)
+
+        all_comments = sorted(all_comments, key=lambda comment: comment["updatedAt"])
```

Comment:
Can you just add the lists?

```suggestion
        all_comments = sorted(
            self.raw["comments"]["nodes"] + self.reviews(),
            key=lambda comment: comment["updatedAt"]
        )
```